projects/module-2/fitlit-part-one-agile.md

```diff
@@ -101,185 +105,244 @@ You should not be hardcoding the date.  You should be using code to find the las
+
+**You are *expected* to test:**
+- All functions that do not update the DOM.  This means everything in your `scripts.js` file that should be tested.
+
+**Remember to test all possible outcomes (happy/sad/etc). Ask yourself:**
+
+- Does the function return anything?
+- Are there different possible outcomes to test based on different arguments being passed in?
+
+**You are *not expected* to test:**
+
+- DOM manipulation / DOM manipulating functions (like `document.querySelector(...)`)
+- Fetch calls
 
 ---
+## Minimum Collaboration and Professionalism Expectations
 
-### Minimum Professionalism Expectations
-* Team collaborates effectively.  Team holds daily check-ins throughout project.
+* Team holds daily standups throughout project.
 * Commits are atomic and frequent, effectively documenting the evolution/progression of the application. There is no more than a 10% disparity in project contributions between teammates.
-* The Project Board is utilized with Github issues and labels.
-* Developer uses PRs from feature branches before adding new code to the main branch.
+* A project board is utilized (and updated throughout the project) with Github issues and labels.
+* Team uses branches, PRs and thorough code reviews to add new code to the main branch.
 * The README is formatted well and at a minimum contains:
   * Overview of project and goals
   * Overview of technologies used, challenges, wins, and any other reflections
   * Screenshots/gifs of your app
+  * List of contributors
+* **Team collaborates effectively to accomplish the shared goal.  Team productively and professionally works through challenges and conflicts to ensure all team members are able to be heard and contribute throughout the project.**  
+  * Instructors are available to offer support and guidance but conversations around what *is* and what *is not* working are expected to be led by the team members themselves.
+
+### Daily Check-Ins
+
+Every work day, your group should do a check-in as a whole group, preferably live over Zoom, and discuss at least these things:
+
+- What is the state of the project so far? (update your GitHub project board if necessary)
+- What is each person working on today?
+- Are there any blockers, and what is your plan to get through the blocker?
+
+Your daily check-in (sometimes called “stand-ups”) schedule should be outline in your DTR so the whole group is aware of the meeting.
+
+---
+
+## Rubric
+
+For the rubric sections below, you will be scored as **Wow**, **Yes** or **Not Yet** depending on whether you have demonstrated competency in that area. Each section lists examples of what types of things we may be looking for as demonstrations of competency. Just as there are many ways to approach code, there are many many ways to demonstate competency.  Thesee are just some examples.
```

Comment:
Tiny typo towards the end of line 285 - extra 'e' in these

Reply:
Good catch, thank you!